tests/ext/otel_config_remapping.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Test OpenTelemetry config remapping
+--ENV--
+OTEL_SERVICE_NAME=service
+OTEL_RESOURCE_ATTRIBUTES=foo=bar,deployment.environment=env,service.name=ignored,xyz=abc,service.version=1.2.3,baz=qux
+OTEL_TRACES_EXPORTER=none
+OTEL_METRICS_EXPORTER=none
+--INI--
+OTEL_SERVICE_NAME=other service
+OTEL_LOG_LEVEL=warn
+OTEL_PROPAGATORS[0]=datadog
+OTEL_PROPAGATORS[1]=b3
```

Comment:
FYI for the propagators, this `"b3"` OTEL_PROPAGATORS should be mapped to the Datadog `"b3 single header"` propagator

Reply:
It looks like this is being done in the product code, so maybe the test snapshot just needs to be updated?

Reply:
yeah, I'll fix it.